### ISSUE TITLE
Cannot use "yarn factor" on Windows

### ISSUE THREAD

**Issue by jozsefsallai** [fictionco/fiction#108]
**Reason for this issue**
The factor cli does not work in Windows due to a broken `factor.cmd`.

**Steps to Reproduce**

1. Create a factor app.
2. Try to run `yarn factor dev` (or any other factor command).

Output:
```
$ yarn factor dev
yarn run v1.13.0
$ D:\Work\factor-test\node_modules\.bin\factor dev
The system cannot find the path specified.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Environment:
 * Windows 10
 * No WSL. Git-bash, but it does the same thing under the native Windows Command Prompt too.
 * Node v10.15.2
 * npm v6.4.1
 * yarn v1.13.0

**Workaround/fix**
The batch script tries to access `/bin/sh.exe` instead of node's executable (which should be in PATH). To fix the issue, replace this in `node_modules/.bin/factor.cmd`:
```batch
@IF EXIST "%~dp0\/bin/sh.exe" (
  "%~dp0\/bin/sh.exe"  "%~dp0\..\@factor\cli\index.js" %*
) ELSE (
  @SETLOCAL
  @SET PATHEXT=%PATHEXT:;.JS;=;%
  /bin/sh  "%~dp0\..\@factor\cli\index.js" %*
)
```
with this:
```batch
@IF EXIST "%~dp0\node.exe" (
  "%~dp0\node.exe"  "%~dp0\..\@factor\cli\index.js" %*
) ELSE (
  @SETLOCAL
  @SET PATHEXT=%PATHEXT:;.JS;=;%
  node  "%~dp0\..\@factor\cli\index.js" %*
)
```

**Comment by arpowers** [comment]
Interesting, thanks for the report. Do you know if it works w `npx`?

Eg `npx factor dev`

I'll have a look around and see how other libs are dealing w this.

**Comment by jozsefsallai** [comment]
`npx` doesn't work either.
```
$ npx factor dev
The system cannot find the path specified.
```

`npx create-factor-app`, on the other hand, did work. It's how I created the project.

> I'll have a look around and see how other libs are dealing w this.

I've checked batch scripts from some of Factor's dependencies and they all have `node` instead of `/bin/sh`. For example, the `atob` package:
```batch
@IF EXIST "%~dp0\node.exe" (
  "%~dp0\node.exe"  "%~dp0\..\atob\bin\atob.js" %*
) ELSE (
  @SETLOCAL
  @SET PATHEXT=%PATHEXT:;.JS;=;%
  node  "%~dp0\..\atob\bin\atob.js" %*
)
```

I had a look at the cli's source code and I couldn't really figure out what causes it to be `/bin/sh` instead of `node`. I'm suspecting it might be an upstream issue?

**Comment by jozsefsallai** [comment]
Actually, it's probably because of the shebang here: https://github.com/fiction-com/factor/blob/353b727a55f75bf11c62bc55d23a7d47b785be86/%40factor/cli/index.js#L1

It should look like this: https://github.com/rvagg/node-errno/blob/master/cli.js#L1

**Comment by arpowers** [comment]
Awesome work. I'll fix and release 1.1.21 by end of day
